lib/KeyboardTrackingViewManager.m

```diff
@@ -428,22 +428,23 @@ - (void)_updateScrollViewInsets
         UIEdgeInsets insets = self.scrollViewToManage.contentInset;
         CGFloat bottomSafeArea = [self getBottomSafeArea];
         CGFloat bottomInset = MAX(self.bounds.size.height, _observingInputAccessoryView.keyboardHeight + _observingInputAccessoryView.height);
-
+        
         CGFloat originalBottomInset = self.scrollIsInverted ? insets.top : insets.bottom;
         CGPoint originalOffset = self.scrollViewToManage.contentOffset;
-
-        bottomInset += (_observingInputAccessoryView.keyboardHeight == 0 ? bottomSafeArea : 0);
+        CGFloat keyboardHeight = _observingInputAccessoryView.keyboardHeight;
+        
+        bottomInset += (keyboardHeight == 0 ? bottomSafeArea : 0);
         if(self.scrollIsInverted)
         {
             insets.top = bottomInset;
         }
         else
         {
-            insets.bottom = bottomInset;
+            insets.bottom = (keyboardHeight == 0 ? 0 : keyboardHeight);
```

Comment:
Could be `insets.bottom = keyboardHeight` ?

Reply:
@chuttam not sure if objective-c allows that syntax

Reply:
Hahahha... sorry, not with `?` at the end. That was part of the question, not the code.

I'll use the "code suggestion" feature next time, to avoid confusion. This change has already been handled in PR review so all good 👍